### ISSUE TITLE
'Stash and reapply' better to stash new files too

### ISSUE THREAD

**Issue by sm-g** [ForkIssues/TrackerWin#212]
Because when I have conflicts after pull with 'rebase instead of merge', I have to remember to not stage all unstaged files after resolving conflicts.

**Comment by DanPristupov** [comment]
Git can't stash new files (until you mean ALL untracked files including ignored ones).

> I have to remember to not stage all unstaged files after resolving conflicts.

What's the problem?

**Comment by sm-g** [comment]
but Stash button have flag "include new files"

> What's the problem?

error prone. Also when there was many conflicts resolved, nice to click "stage all" button

**Comment by DanPristupov** [comment]
> but Stash button have flag "include new files"

You mean the Stash dialog? I think it doesn't. There is '**stage** new files' option, which is not the same.

**Comment by sm-g** [comment]
Yes. I want to not see all uncommited files during conflict resolving after pull.
